.github/workflows/manual-snapshot-publisher.yml

```
@@ -0,0 +1,873 @@
+
+          matrix="${matrix%,}]}"
+
+          echo "containers-matrix=$matrix" >> $GITHUB_OUTPUT
+          echo "container-count=${#containers[@]}" >> $GITHUB_OUTPUT
+          echo "selected-containers=$container_list" >> $GITHUB_OUTPUT
+
+          echo "📊 Matrix: $matrix"
+
+      - name: 🏷️ Generate Tag Configuration
+        id: generate-tag
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:73:32: Double quote to prevent globbing and word splitting

(shellcheck)